llvm/lib/CodeGen/SelectionDAG/SelectionDAGBuilder.cpp

```diff
@@ -6706,6 +6707,9 @@ void SelectionDAGBuilder::visitIntrinsicCall(const CallInst &I,
     case Intrinsic::fabs:      Opcode = ISD::FABS;       break;
     case Intrinsic::sin:       Opcode = ISD::FSIN;       break;
     case Intrinsic::cos:       Opcode = ISD::FCOS;       break;
+    case Intrinsic::tan:
+      Opcode = ISD::FTAN;
+      break;
```

Comment:
indentation

Reply:
I can't, clan format github action will fail.

Reply:
I think we need to fix this once and for all by turning all of these horrible case, assign, break patterns into separate helper functions that just return the value. This happens on every DAG opcode patch